composer.json

```diff
@@ -18,7 +18,8 @@
         { "name": "Andreas Braun", "email": "andreas.braun@mongodb.com", "role": "Leader" },
         { "name": "Jérôme Tamarelle", "email": "jerome.tamarelle@mongodb.com", "role": "Maintainer" },
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com", "role": "Maintainer" },
-        { "name": "Jens Segers", "homepage": "https://jenssegers.com", "role": "Creator" }
+        { "name": "Jens Segers", "homepage": "https://jenssegers.com", "role": "Creator" },
+        { "name": "Pauline Vos", "email": "pauline.vos@mongodb.com", "role": "Maintainer" }
```

Comment:
Wasn't sure if this was alphabetical order or by role 🤔

Reply:
I'd say in order of importance, put your name after Andreas 😉